azure-spring-maven-plugin/src/main/java/com/microsoft/azure/maven/spring/configuration/Deployment.java

```diff
@@ -34,6 +35,10 @@ public Integer getInstanceCount() {
         return instanceCount;
     }
 
+    public String getDeploymentName() {
+        return deploymentName;
```

Comment:
user needs to specify the deploymentName in pom? how about the config goal? what's the default deployment name? also there needs some documentation in UX spec.